features/admin_center_dashboard.feature

```diff
@@ -19,11 +19,7 @@ Feature: Dashboard
     Given I am logged in with email "affiliate_manager_with_no_affiliates@fixtures.org"
     When I go to the sites page
     Then I should see "Add a New Site"
-
-  Scenario: Viewing a site without logging in
-    When I go to the usagov's Dashboard page
-    Then I should see "Login" button
-
```

Comment:
@MothOnMars and I discussed and agreed this is already being tested in rspec

Reply:
👍